tests/pgautofailover_utils.py

```diff
@@ -115,11 +115,11 @@ def pg_createcluster(self, datadir, port=5432):
 
         print("%s" % " ".join(chmod_command))
 
-        chmod_proc = vnode.run(chmod_command)
-        out, err = chmod_proc.communicate(timeout=COMMAND_TIMEOUT)
-        if chmod_proc.returncode > 0:
-            raise Exception("chmod failed, out: %s\n, err: %s" %
-                            (out, err))
+        with vnode.run(chmod_command) as chmod_proc:
+            out, err = chmod_proc.communicate(timeout=COMMAND_TIMEOUT)
+            if chmod_proc.returncode > 0:
+                raise Exception("chmod failed, out: %s\n, err: %s" %
+                                (out, err))
```

Comment:
Same as before, looks like a single call to `self.vnode.run_and_wait()` now, right?